mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java

```diff
@@ -183,10 +183,10 @@ private ByteBuffer toByteBuffer(InputStream stream) throws IOException {
 
     public CompletableFuture<Stream<Pair<MessageWithoutAttachment, Stream<MessageAttachmentRepresentation>>>> retrieveMessages(List<ComposedMessageIdWithMetaData> messageIds, FetchType fetchType, Optional<Integer> limit) {
         return retrieveRows(messageIds, fetchType, limit)
-                .thenApply(resultSet -> toto(messageIds, fetchType, resultSet));
+                .thenApply(resultSet -> toMessagesWithoutAttachements(messageIds, fetchType, resultSet));
     }
 
-    private Stream<Pair<MessageWithoutAttachment, Stream<MessageAttachmentRepresentation>>> toto(List<ComposedMessageIdWithMetaData> messageIds, FetchType fetchType, ResultSet resultSet) {
+    private Stream<Pair<MessageWithoutAttachment, Stream<MessageAttachmentRepresentation>>> toMessagesWithoutAttachements(List<ComposedMessageIdWithMetaData> messageIds, FetchType fetchType, ResultSet resultSet) {
```

Comment:
I don't understand the function name: you tell "withoutAttachment" but you are returning a pair where the second element is a Stream of MessageAttachmentRepresentation. It is expected?